Sources/WalletConnectNetworking/Networking.swift

```diff
@@ -8,13 +8,20 @@ public class Networking {
     }
 
     public static var interactor: NetworkingInteractor = {
-        guard let _ = Networking.config else {
+        guard let config = Networking.config else {
             fatalError("Error - you must call Networking.configure(_:) before accessing the shared instance.")
         }
 
-        return NetworkingClientFactory.create(relayClient: Relay.instance)
+        return NetworkingClientFactory.create(relayClient: Relay.instance, groupIdentifier: config.groupIdentifier)
     }()
 
+    public static var groupIdentifier: String {
+        guard let config = Networking.config else {
```

Comment:
we're copy and pasting fatal error. May be create `private static var config: Config` and throw fatal error only there?